src/main/java/com/programs/Programs.java

```diff
@@ -11,13 +11,13 @@ public static void main(String[] args){
 
```

Comment:
remove this class, now on every method should be asserted using unit test only

Reply:
I have removed the class.